app/mailers/user_mailer.rb

```diff
@@ -3,9 +3,12 @@ class UserMailer < ApplicationMailer
   default from: "notifications@example.com"
 
   def review_alert_email(user)
+
     @user = user
-    @url  = "http://www.chinatown-aces.herokuapp.com"
-    mail(to: @user.email, subject: "Information regarding your business")
+    @url  = 'http://www.chinatown-aces.herokuapp.com'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.